exchanges/kraken/kraken_websocket.go

```diff
@@ -606,7 +606,7 @@ func (k *Kraken) wsProcessTrades(channelData *WebsocketChannelData, data []inter
 // Then sends to appropriate fun
 func (k *Kraken) wsProcessOrderBook(channelData *WebsocketChannelData, data map[string]interface{}) error {
 	if fullAsk, ok := data["as"].([]interface{}); ok {
-		fullBids := data["as"].([]interface{})
+		fullBids := data["bs"].([]interface{})
```

Comment:
Nice pickup @vazha! Like codelingo mentions, it would be good to protect the code a little more (no matter how unlikely the scenario of an orderbooksnapshot not returning bids), perhaps something more along these lines:
```
        askSnapshot, askSnapshotExists := data["as"].([]interface{})
	bitSnapshot, bidSnapshotExists := data["bs"].([]interface{})
	if askSnapshotExists || bidSnapshotExists {
		err := k.wsProcessOrderBookPartial(channelData, askSnapshot, bitSnapshot)
		if err != nil {
			return err
		}
	} else {
		asks, asksExist := data["a"].([]interface{})
		bids, bidsExist := data["b"].([]interface{})
		if asksExist || bidsExist {
                      ....
```

Reply:
Great pickup indeed!